File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--protect-parts-of-documents-by-using-content-controls_1.vb
    Dim deletableControl As Microsoft.Office.Tools.Word.RichTextContentControl
    Dim editableControl As Microsoft.Office.Tools.Word.RichTextContentControl

    Private Sub AddProtectedContentControls()
        Me.Paragraphs(1).Range.InsertParagraphBefore()
        Dim range1 As Word.Range = Me.Paragraphs(1).Range

        deletableControl = Me.Controls.AddRichTextContentControl(range1, _
            "deletableControl")
        deletableControl.PlaceholderText = "You can delete this control, " & _
            "but you cannot edit it"
        deletableControl.LockContents = True

        range1.InsertParagraphAfter()
        Dim range2 As Word.Range = Me.Paragraphs(2).Range

        editableControl = Me.Controls.AddRichTextContentControl(range2, _
            "editableControl")
        editableControl.PlaceholderText = "You can edit this control, " & _
            "but you cannot delete it"
        editableControl.LockContentControl = True
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--protect-parts-of-documents-by-using-content-controls_1.cs
        private Microsoft.Office.Tools.Word.RichTextContentControl deletableControl;
        private Microsoft.Office.Tools.Word.RichTextContentControl editableControl;

        private void AddProtectedContentControls()
        {
            this.Paragraphs[1].Range.InsertParagraphBefore();
            Word.Range range1 = this.Paragraphs[1].Range;

            deletableControl = this.Controls.AddRichTextContentControl(range1,
                "deletableControl");
            deletableControl.PlaceholderText = "You can delete this control, " +
                "but you cannot edit it";
            deletableControl.LockContents = true;

            range1.InsertParagraphAfter();
            Word.Range range2 = this.Paragraphs[2].Range;

            editableControl = this.Controls.AddRichTextContentControl(range2,
                "editableControl");
            editableControl.PlaceholderText = "You can edit this control, " +
                "but you cannot delete it";
            editableControl.LockContentControl = true;
        }